sql/updates/reverse-dev.sql

```diff
@@ -2,3 +2,88 @@ DROP VIEW _timescaledb_internal.hypertable_chunk_local_size;
 DROP FUNCTION _timescaledb_internal.relation_size(relation REGCLASS);
 DROP INDEX _timescaledb_catalog.chunk_constraint_dimension_slice_id_idx;
 CREATE INDEX chunk_constraint_chunk_id_dimension_slice_id_idx ON _timescaledb_catalog.chunk_constraint (chunk_id, dimension_slice_id);
+
+DO
+$$
+DECLARE
+    caggs_finalized TEXT;
+    caggs_count INTEGER;
+BEGIN
+    SELECT
+        string_agg(format('%I.%I', user_view_schema, user_view_name), ', '),
+        count(*)
+    INTO
+        caggs_finalized,
+        caggs_count
+    FROM
+        _timescaledb_catalog.continuous_agg
+    WHERE
+        finalized IS TRUE;
+
+    IF caggs_count > 0 THEN
+        RAISE EXCEPTION 'Downgrade is not possible because there are % continuous aggregates using the finalized form: %', caggs_count, caggs_finalized
+            USING HINT = 'Remove the corresponding continuous aggregates manually before downgrading';
+    END IF;
+END;
+$$
+LANGUAGE 'plpgsql';
+
+DROP VIEW IF EXISTS timescaledb_information.hypertables;
+DROP VIEW IF EXISTS timescaledb_information.continuous_aggregates;
+
+ALTER TABLE _timescaledb_catalog.continuous_agg
+    DROP COLUMN finalized;
```

Comment:
need to rebuild the table

Reply:
but this is exactly what I'm doing below

Reply:
yes  i saw that not sure why you are dropping the column here when you rebuild the table below